ncm-spma/src/main/pan/components/spma/yum/schema.pan

```diff
@@ -33,8 +33,21 @@ type spma_yum_plugins = {
     "priorities" ? spma_yum_plugin_priorities
 };
 
+@documentation{
+    Configuration options for yum.conf.
+    The cleanup_on_remove, obsoletes, reposdir and pluginpath are set internally.
+}
+type spma_yum_conf = {
+    "exclude" ? string[]
+    "installonly_limit" ? long(0..) = 3
+    "keepcache" ? boolean
+    "retries" ? long(0..) = 10
+    "timeout" ? long(0..) = 30
+};
+
 type component_spma_yum = {
     "userpkgs_retry" : boolean = true
     "fullsearch" : boolean = false
     "plugins" ? spma_yum_plugins
+    "yumconf" ? spma_yum_conf
```

Comment:
the schema is shared with `yum` and `ips` (and `yumng`), and the options are not generic enough imho. but i'll change it to whatever is most suitable ;)